proposals/0002.md

```diff
@@ -0,0 +1,281 @@
+* Title: Structured error handling in Rust
+* Champion: @CodeSandwich
+* Conversation (optional): [thread on radicle.community](https://radicle.community/t/proposal-structured-error-handling-in-rust/179)
+* References:
+  * [The previous error handling proposal](https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0001.md)
```

Comment:
FYI: you can use anchor references. If you want to look at the previous proposal I used them. It means you don't have to repeat the link but instead, the anchor.